neuroinformatics/hbp_prov/hbp-prov-schema-v3.1.json

```diff
@@ -0,0 +1,959 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "description": "schema for a the json that describes the registration activity.",
+  "version": "3.0",
```

Comment:
As discussed previously in person before, I believe the various instances of the schema have to be publicly available on the web so that a user can grab the schema and validate an instance of HBP-PROV. I am fully aware that this is making tracking of the history in git split across multiple files, but unless I am being offered an other constructive alternative (incl. devops aspects of deploying schemas publicly) I propose we stick with that scheme.